### PR TITLE
chart: add and update manifests for recovery-backend

### PR DESCRIPTION
**Pull request by derekbit**
https://github.com/longhorn/longhorn/issues/2293

Signed-off-by: Derek Su [derek.su@suse.com](mailto:derek.su@suse.com)

### CODE REVIEW

File: chart/templates/services.yaml [hunk 1]
```diff
@@ -34,6 +34,23 @@ spec:
...
+  namespace: {{ include "release_namespace" . }}
+spec:
+  type: ClusterIP
+  sessionAffinity: ClientIP
+  selector:
+    app: longhorn-recovery-backend
+  ports:
+  - name: recovery-backend
+    port: 9600
+    targetPort: recov-backend
```
Comment (innobead) [inline chart/templates/services.yaml:50]: Probably we can just the same name `recovery-backend`as the service name.

**Review by innobead — COMMENTED** [review]
LGTM. Just one comment for the consistency.

File: chart/templates/services.yaml (hunk 1 above)
Comment (derekbit) [reply chart/templates/services.yaml:50]: targetPort must be no more than 15 characters. how about using `recov-backend`?

Comment (innobead) [reply chart/templates/services.yaml:50]: I see. This is why the name is used on purpose. Let's just keep it as is.

**Review by innobead — DISMISSED** [review]

**Comment by derekbit** [comment]
Waiting for https://github.com/longhorn/longhorn-share-manager/pull/36

**Comment by derekbit** [comment]
e2e
test_rwx.py
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/1934/

**Review by innobead — APPROVED** [review]

**Pull request merged**